binding/python/test_porcupine.py

```diff
@@ -73,7 +46,50 @@ def test_process_multiple(self):
 
         porcupine.delete()
 
-        self.assertEqual(results, [6, 0, 1, 2, 3, 4, 5, 6, 7])
+        self.assertEqual(results, ground_truth)   
+
+    def test_model_en(self):
+        lang = 'en'
+
+        self.run_porcupine(lang = lang,
```

Comment:
though this is a bit pedantic, we tend to favour more verbose variable names in our public code, so `lang` -> `language`